dataactvalidator/scripts/loader_utils.py

```diff
@@ -19,7 +19,7 @@
 def clean_col_names(field):
     """Define some data-munging functions that can be applied to pandas
     dataframes as necessary"""
-    return str(field).lower().strip().replace(" ", "_").replace(",", "_")
+    return str(field).lower().strip().replace(" ", "_").replace(",", "_").replace("/", "_")
```

Comment:
I'm a little worried about this change cause I think a lot uses this but the fact that tests passed probably means it's fine.